VideoLocker/res/values/styles.xml

```diff
@@ -106,6 +105,10 @@
         <item name="android:background">@color/edx_brand_primary_base</item>
     </style>
 
+    <style name="course_dashboard_separator_style" parent="separator_style">
```

Comment:
2 in total:
1 - `course_dashboard_separator_style` uses `edx_grayscale_neutral_light` color
2 - `course_header_separator_style` uses `edx_brand_primary_base` color